interfaces/console/src/app/console/subscribers/page.tsx

```diff
@@ -274,6 +220,18 @@ const Page = () => {
       });
     },
   });
+  const { data: networkList } = useGetNetworksQuery({
+    fetchPolicy: 'cache-and-network',
+
+    onError: (error) => {
+      setSnackbarMessage({
+        id: 'networks-msg',
+        message: error.message,
+        type: 'error' as AlertColor,
+        show: true,
+      });
+    },
+  });
```

Comment:
we are not showing networks list on subscriber data table anymore. Why we're fetching networks now?